app/src/main/java/de/westnordost/streetcomplete/quests/leaf_detail/AddTreeLeafType.kt

```diff
@@ -0,0 +1,34 @@
+package de.westnordost.streetcomplete.quests.leaf_detail
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.elementfilter.toElementFilterExpression
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.OUTDOORS
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddTreeLeafType : OsmFilterQuestType<TreeLeafType>() {
+    override val elementFilter = """
+        nodes with natural = tree and !leaf_type
```

Comment:
Should not be asked if another tag already gives a more precise information about the type of tree

Reply:
```
        and !species:wikidata and !species:wikipedia and !genus:wikidata and !genus:wikipedia
        and !genus and !species
```

anything else?

Reply:
genus:de etc?

Check out NameAndLocationLabel, there is some tree taxon stuff

Reply:
should be done now